Portal/src/Datahub.Functions/Program.cs

```diff
@@ -61,6 +84,19 @@
         services.AddScoped<IDateProvider, DateProvider>();
         services.AddScoped<EmailValidator>();
 
+        services.AddAzureClients(builder =>
+        {
+            builder.AddClient<ArmClient, ArmClientOptions>(options =>
+            {
+                options.Diagnostics.IsLoggingEnabled=true;
+                var tenantId = config.GetValue<string>("TENANT_ID");
+                var clientId = config.GetValue<string>("FUNC_SP_CLIENT_ID");
+                var clientSecret = config.GetValue<string>("FUNC_SP_CLIENT_SECRET");
+                var creds = new ClientSecretCredential(tenantId, clientId, clientSecret);
+                var client = new ArmClient(creds);
+                return client;
+            });
+        });
```

Comment:
What's the difference between the two `services.AddAzureClients` calls?

Reply:
None, this is a mistake on my side, will remove one of them